sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/model/NativeException.java

```diff
@@ -0,0 +1,10 @@
+package com.microsoft.appcenter.crashes.model;
+
+public class NativeException extends RuntimeException {
```

Comment:
does not seem native exception to me, can you explain?

Reply:
It only represents the NDK exception, but isn't an actual one. This is only used after a breakpad minidump has been generated.